internal/pkg/v2/bootstrap/container/container.go

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package container
+
+import (
+	"github.com/edgexfoundry/edgex-go/internal/core/data/config"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/controller/errorconcept"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/infrastructure/interfaces"
+
+	"github.com/edgexfoundry/go-mod-bootstrap/bootstrap/container"
+	"github.com/edgexfoundry/go-mod-bootstrap/di"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/metadata"
+
+	"github.com/edgexfoundry/go-mod-messaging/messaging"
+)
+
+type Container struct {
+	ConfigClient       *config.ConfigurationStruct
+	DBClient           interfaces.DBClient
+	ErrorHandlerClient *errorconcept.Handler
+	EventChan          chan<- interface{}
+	LoggingClient      logger.LoggingClient
+	MessageClient      messaging.MessageClient
+	MetadataClient     metadata.DeviceClient
+}
+
+var BootstrapContainer *di.Container
```

Comment:
Please do not use global variables. Use structs and constructor and receiver functions to avoid global variables.
example:
```
type MyThing struct {
  BootstrapContainer *di.Container
  RegistryClients Container
}

func NewMyThing() MyThing {
   return MyThing{}
}

func (thing MyThing) Init() {
  ...
}
```

Reply:
Hi Lenny, the above code has been refactor in [internal/core/data/v2/bootstrap/container/serviceinjector.go](https://github.com/edgexfoundry/edgex-go/blob/9bb6161d8711a2382a3e6df77ea55c1240982a91/internal/core/data/v2/bootstrap/container/serviceinjector.go),  could you please review the change? Thanks.